shenfun_elasticity/solver/elastic_problem.py

```diff
@@ -69,16 +68,17 @@ def _cleanup_boundary_conditions(self):
                     if len(bc['left']) > 0:
                         single_bc = bc['left'][0]
                         index = 0
-                    else:
+                    elif len(bc['left']) > 0:
```

Comment:
I think this sould be 'right' instead of 'left'.

Reply:
Yes that's copy-paste error